src/components/ControllerTodo.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { addTodo, clearTodo, sortByMin, sortByMax, toggleComplete, removeTodo } from '../store/todosSlice'
+
+
+
+
+export default function ControllerTodo({renderSearch, updateTodo}) {
+  const [text, setText] = useState('')
+  const [search, searchText] = useState('')
+  const state = useSelector(state => state)
```

Comment:
```suggestion
  const state = useSelector(state => state.todos)
```
тебе тут надо конкретный слайс а не всё